### PR TITLE
ElevenLabs-DotNet 2.0.3

### PR DESCRIPTION
**Pull request by ed-sparkes**
- Fixed text to speech streaming

### CODE REVIEW

File: ElevenLabs-DotNet/ElevenLabs-DotNet.csproj [hunk 1]
```diff
@@ -14,8 +14,10 @@
     <Company>RageAgainstThePixel</Company>
     <Copyright>2023</Copyright>
     <PackageTags>ElevenLabs, AI, ML, Voice, TTS</PackageTags>
-    <Version>2.0.2</Version>
-    <PackageReleaseNotes>Version 2.0.2
+    <Version>2.0.3</Version>
+    <PackageReleaseNotes>Version 2.0.3
+- Fix streaming issue
```
Comment (StephenHodgson) [inline ElevenLabs-DotNet/ElevenLabs-DotNet.csproj:19]:
```suggestion
- Fixed text to speech streaming
```

**Comment by ed-sparkes** [comment]
Hmm whats the deal with the tests? lots fail due to invalid api key? i didnt change anything there

**Comment by StephenHodgson** [comment]
> Hmm @StephenHodgson whats the deal with the tests? lots fail due to invalid api key? i didnt change anything there

Because it is a fork. I didn't give secret permissions to them. Which is fine. I will validate locally first anyway. About to push some tweaks.

File: ElevenLabs-DotNet/TextToSpeech/TextToSpeechEndpoint.cs [hunk 2]
```diff
@@ -87,11 +87,13 @@ public async Task<VoiceClip> TextToSpeechAsync(string text, Voice voice, VoiceSe
             {
                 parameters.Add(OptimizeStreamingLatencyParameter, optimizeStreamingLatency.ToString());
             }
-            var postRequest = new HttpRequestMessage(HttpMethod.Post, GetUrl($"/{voice.Id}{(partialClipCallback == null ? string.Empty : "/stream")}", parameters))
-            {
-                Content = payload,
-            };
-            var response = await Api.Client.SendAsync(postRequest, partialClipCallback == null ? HttpCompletionOption.ResponseContentRead :  HttpCompletionOption.ResponseHeadersRead, cancellationToken);
+
+            using var postRequest = new HttpRequestMessage(HttpMethod.Post, GetUrl($"/{voice.Id}{(partialClipCallback == null ? string.Empty : "/stream")}", parameters));
```
Comment (ed-sparkes) [inline ElevenLabs-DotNet/TextToSpeech/TextToSpeechEndpoint.cs:91]: why the using here? without a scope?

Comment (StephenHodgson) [reply ElevenLabs-DotNet/TextToSpeech/TextToSpeechEndpoint.cs:91]: `HttpRequestMessage` implements `IDisposable`

**Pull request merged**